package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "remote-content",
-  "version": "1.0.2",
+  "version": "1.0.3",
```

Comment:
Oh yeah, I just noticed this. Don't bump the package number in a pull request. I'll take care of it when it's published.

Reply:
Apologies for that.  I've submitted another commit.

Reply:
Would the proxy support be considered a new feature or a patch?